packages/rich-text/package.json

```diff
@@ -49,25 +49,25 @@
     "@contentful/rich-text-plain-text-renderer": "^16.0.4",
     "@contentful/rich-text-types": "16.1.0",
     "@popperjs/core": "^2.11.5",
-    "@udecode/plate-basic-marks": "18.15.0",
-    "@udecode/plate-break": "18.15.0",
-    "@udecode/plate-core": "18.15.0",
-    "@udecode/plate-list": "18.15.0",
-    "@udecode/plate-paragraph": "18.15.0",
-    "@udecode/plate-reset-node": "18.15.0",
-    "@udecode/plate-select": "18.15.0",
-    "@udecode/plate-serializer-docx": "18.15.0",
-    "@udecode/plate-serializer-html": "18.15.0",
-    "@udecode/plate-table": "18.15.0",
-    "@udecode/plate-trailing-block": "18.15.0",
+    "@udecode/plate-basic-marks": "23.7.0",
+    "@udecode/plate-break": "23.7.0",
+    "@udecode/plate-core": "23.6.0",
+    "@udecode/plate-list": "23.7.0",
+    "@udecode/plate-paragraph": "23.7.0",
+    "@udecode/plate-reset-node": "23.7.0",
+    "@udecode/plate-select": "23.7.0",
+    "@udecode/plate-serializer-docx": "23.7.0",
+    "@udecode/plate-serializer-html": "23.7.0",
+    "@udecode/plate-table": "23.7.0",
+    "@udecode/plate-trailing-block": "23.7.0",
     "fast-deep-equal": "^3.1.3",
     "is-hotkey": "^0.2.0",
     "is-plain-obj": "^3.0.0",
     "react-popper": "^2.3.0",
-    "slate": "0.80.0",
+    "slate": "0.94.1",
     "slate-history": "0.66.0",
-    "slate-hyperscript": "0.67.0",
-    "slate-react": "0.80.0"
+    "slate-hyperscript": "0.77.0",
+    "slate-react": "0.98.3"
```

Comment:
This will result in some issues - `@udecode/plate-core` depends on `slate >=0.94.0`. The conflict in slate dependencies between the field-editors packages and experience-components currently adds nearly 700kb to the web app's bundle size.
We should either try and align the versions between the two, or adopt proper semver specifiers if Slate's release process supports it.

Reply:
Great callout! we will address this in a follow up